lib/classes/PluginManager.js

```diff
@@ -215,6 +220,38 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPaths() {
+    const fs = require('fs');
+
+    const pluginsPaths = this.localPluginsPaths.map(({ entryFileRealPath, originalPath }) => {
+      // Scenario 1: a file was specified WITH extension, eg "./folder/my-plugin.js"
+      if (originalPath.endsWith('.js')) {
```

Comment:
While it's a corner case, user may use folder that ends with `.js`.

Solid approach would be to probably compare something as:

```javascript
if (path.resolve(serviceDir, inputPath) === realPath) // assume realPath
```